### PR TITLE
New version: Streetwriters.Notesnook version 2.6.16

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/142700)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Installer https://github.com/streetwriters/notesnook/releases/download/v2.6.16/notesnook_win_arm64_portable.exe
 Scan request id: '69302574-c6d5-4805-b990-b5ff80d95bab'
 
 Installer https://github.com/streetwriters/notesnook/releases/download/v2.6.16/notesnook_win_x64_portable.exe
 Scan request id: '49b471d8-f954-4911-83ec-b590008eb3ca'
 
 Specified hash doesn't match.
 Actual hash:   20F12C69195A73FC56BCE80B761E14CF2A3F95B38B1623C72565973B8A07E555
 Expected hash: edecae7852d622179f6dafcefcd735a2e3ba1b1817024016cc06690e884a7953
 Specified hash doesn't match.

(Automated response - build 861.)

File: manifests/s/Streetwriters/Notesnook/2.6.16/Streetwriters.Notesnook.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v2.6.16/notesnook_win_x64_portable.exe
+  InstallerSha256: CE42080DCFF5D55601E5A8D4269E19759809ABB133FCA1D373B4AD4B21ACFFCC
+- Architecture: arm64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v2.6.16/notesnook_win_arm64.exe
+  InstallerSha256: EDECAE7852D622179F6DAFCEFCD735A2E3BA1B1817024016CC06690E884A7953
+- Architecture: arm64
+  InstallerType: portable
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v2.6.16/notesnook_win_arm64_portable.exe
+  InstallerSha256: EDECAE7852D622179F6DAFCEFCD735A2E3BA1B1817024016CC06690E884A7953
```
Comment (stephengillie) [inline manifests/s/Streetwriters/Notesnook/2.6.16/Streetwriters.Notesnook.installer.yaml:27]:
```suggestion

```

(Automated response - build 861.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-03-06T15:30:12.9820417Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][2] Description: Missing required property 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #2 in array. Error context: <root>[Installers][3] Description: Missing required property 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #3 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: Streetwriters.Notesnook.installer.yaml Manifest Error: Required field missing. [InstallerSha256] Manifest Error: Required field missing. [InstallerSha256]

(Automated response - build 861.)

File: manifests/s/Streetwriters/Notesnook/2.6.16/Streetwriters.Notesnook.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- Architecture: x64
+  InstallerType: portable
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v2.6.16/notesnook_win_x64_portable.exe
+  InstallerSha256: CE42080DCFF5D55601E5A8D4269E19759809ABB133FCA1D373B4AD4B21ACFFCC
+- Architecture: arm64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v2.6.16/notesnook_win_arm64.exe
+- Architecture: arm64
+  InstallerType: portable
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v2.6.16/notesnook_win_arm64_portable.exe
```
Comment (stephengillie) [inline manifests/s/Streetwriters/Notesnook/2.6.16/Streetwriters.Notesnook.installer.yaml:25]: Hi @Exorcism0666,

Could installer hashes be added for these 2 installers?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**